openff/bespokefit/cli/executor/list.py

```diff
@@ -49,11 +61,19 @@ def list_cli():
 
     settings = current_settings()
 
+    status_url = (
+        None
+        if status_filter is None
+        else status_filter.replace("success", "complete").replace("errored", "complete")
```

Comment:
Just to make sure I understand, we keep errored tasks in the complete queue so we have to query that queue and then filter if the actual status of the task is errored?

Reply:
slightly picky can I add a comment saying this as I am sure I will forget in future!

Reply:
Absolutey! I've added one in fabcedc